synapse/appservice/api.py

```diff
@@ -321,16 +321,18 @@ def _serialize(
             serialize_event(
```

Comment:
Part of me thinks we should be consistent and use `EventSerializer` here, but it doesn't really matter unless we might bundled aggregations. I dislike that this is two separate methods though.

Reply:
sounds like it might be worthwhile as an extra pass if you have the time and it looks sensible as you're doing it?